.github/actions/get-runner/action.yml

```diff
@@ -52,31 +57,60 @@ runs:
             echo "[INFO] You do NOT have ROOT permission to set system config."
             echo "       The frequency governor is ${scaling_governor}."
         fi
+    - name: Info for tests
+      shell: bash -xe {0}
+      id: tests
+      run: |
+        # cpu number
         cpu_num="$(lscpu |grep -E 'Core\(s\) per socket:|Socket\(s\):' |awk 'BEGIN{sum=1}{sum*=$NF}END{printf sum}')"
-        xpu_num="$(clinfo --list |awk 'BEGIN{gpu=0;}{
-          if(gpu==1 && $0~/Platform/){gpu=0}; if(gpu==1){print $0}; if($0~/Platform.*Graphics/){gpu=1}
-        }' |wc -l)"
-        cpus_per_xpu="$(echo |awk -v c="${cpu_num}" -v x="${xpu_num}" '{printf c/x}')"
+        # available gpu card number
```

Comment:
These changes were not here yesterday. Are they absolutely necessary? I also see that few other workloads besides accelerate and transformers got changed. Is this necessary as well?